### PR TITLE
TVB-2122: Parallelize library tests

### PR DESCRIPTION
**Pull request by VinczeRobert**
I found a pretty easy way to parallelize the library tests. We need a plug-in called pytest-xdist and we need to add the -n NUMBER_OF_CORES when running the pytest command. Using the -n auto option it will run the tests on as many cores as your machine has.

Running the library tests normally took 580.90 s on my machine.
Using parallelization (I have 8 cores) they ran in 173.20, which means more than 3 times faster.

### CODE REVIEW

**Review by maedoc — APPROVED** [review]
This is how the library tests are running on the GitHub CI runner, with the `pytest-xdist` plugin, and it's worth using.  A few caveats, (a) benchmarks no longer run, so if you want those, you need to do a [separate pytest run](https://github.com/the-virtual-brain/tvb-root/blob/master/.github/workflows/lib-tests.yml#L67) for that and (b) this should be used with caution if your testing VM has not been provisioned appropriately ( 1 or 2 gb mem / vcpu) mainly because of surface simulations.

File: tvb_build/docker/Dockerfile-build [hunk 1]
```diff
@@ -24,7 +24,7 @@ USER root
 RUN conda update -n base -c defaults conda
 
 RUN conda create -y --name tvb-docs python=3.7 nomkl numba scipy numpy networkx scikit-learn cython pip numexpr psutil
-RUN conda install -y --name tvb-docs pytest pytest-cov pytest-benchmark pytest-mock matplotlib-base
+RUN conda install -y --name tvb-docs pytest pytest-cov pytest-benchmark pytest-mock pytest-xdist matplotlib-base
```
[the remark this answers is not in the document]
Comment (VinczeRobert) [reply tvb_build/docker/Dockerfile-build:27]: Then why are the other pytest modules included (pytest pytest-cov etc. )?

[the remark this answers is not in the document]
Comment (maedoc) [reply tvb_build/docker/Dockerfile-build:27]: probably just copy pasta

[the remark this answers is not in the document]
Comment (VinczeRobert) [reply tvb_build/docker/Dockerfile-build:27]: Ok, I removed that line. I guess we don't need matlab as well.

**Review by liadomide — APPROVED** [review]

**Pull request merged**